app/file-upload/server/lib/FileUpload.js

```diff
@@ -265,6 +265,40 @@ export const FileUpload = {
 		return result;
 	},
 
+	createImageThumbnail(file) {
```

Comment:
If the original image/file is smaller than the configured thumbnail sizes, I think we should skip and go through this process of creating a new image.

Reply:
Done!